Makefile

```diff
@@ -132,6 +132,10 @@ k8s-migrate:
 	$(shell kubectl --namespace ${DEIS_APP} get pods | grep ${DEIS_APP}-cmd | awk '{print $$1}') \
 	python manage.py migrate
 
+render-k8s-templates:
+	k8s/api-svc.yaml.template.sh > k8s/api-svc.yaml
+	rm k8s/api-svc.yaml.template.sh
```

Comment:
I wasn't sure if kubectl would ignore unsupported filenames off the top of my head so I added the `rm` for the initial iteration with the plan of removing it later if it turned out not to be necessary. Thanks for testing @escattone; I'll go ahead and remove it now.